packages/api/src/api/helix/chat/HelixChatChatter.ts

```diff
@@ -0,0 +1,52 @@
+import { Enumerable } from '@d-fischer/shared-utils/lib';
```

Comment:
Fails at runtime due to `exports` definition.

Reply:
Autoreference strikes again. Fixed.